plugins/system/kernel_same_page_merging

```diff
@@ -1,6 +1,3 @@
-#!/bin/bash
```

Comment:
Are you sure, that you want to remove the shebang? This should cause an execution failure, or?

Reply:
Ah - somehow I looked at an old state of the code. Sorry!